### ISSUE TITLE
Reverse proxy issue for web version

### ISSUE THREAD

**Issue by aibosss** [YunoHost-Apps/immich_ynh#80]
Mobile app works trough the https reverse proxy url, 
http://127.0.0.1:3001 on the server works
But WEB/desktop access trough the https reverse proxy url stalls right after the Immich login, blank page after login.

As for context, all the other apps installed trough yunohost with the reverse proxy url work, including nextcloud and many others.

I'm not sure which logs you need, haven't found anything wrong myself in the logs or anywhere

**Comment by ewilly** [comment]
Use the domain you choose at install eg. Https://myimmichdomain.tld

Le 9 août 2024 17:56:11 GMT+02:00, aibosss ***@***.***> a écrit :

**Comment by aibosss** [comment]
Already in use by Synapse which only allow 1 app there. So my tests were with subdomains.

**Comment by aibosss** [comment]
I guess I should add a root domain and try ? =)

**Comment by aibosss** [comment]
So I used a root domain, not the main one, but still a root domain, and the problem is the same..
![image](https://github.com/user-attachments/assets/ab85d948-5ea6-4a65-a80f-5da88734e8b4)

**Comment by aibosss** [comment]
And there are the cooklies that get set on login
![image](https://github.com/user-attachments/assets/71007b1f-6bf0-4881-8989-49ae239757a3)

**Comment by ewilly** [comment]
why the email field seems empty ?

**Comment by aibosss** [comment]
just as to show

**Comment by aibosss** [comment]
Yes I'm a technician myself, and I tried with diff browsers. chromium based and firefox based
You can close issue as I'm already on another platform for Immich because of that problem

**Comment by aibosss** [comment]
You were right, everything was working. Problem was file permissions with my current yunohost users ssowat..
Created a new user, and it worked magically..

**Comment by aibosss** [comment]
Tried on new debian 12 install and works perfect, thanks
